src/community/gwc-sqlite/src/main/java/org/geoserver/gwc/web/blob/SqliteMultipartFilter.java

```diff
@@ -0,0 +1,22 @@
+package org.geoserver.gwc.web.blob;
+
+import org.geoserver.filters.GeoServerFilter;
+import org.springframework.web.multipart.support.MultipartFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class SqliteMultipartFilter extends MultipartFilter implements GeoServerFilter {
+    @Override
+    protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws ServletException, IOException {
+        String path = ((HttpServletRequest) request).getRequestURI();
+        if (path.contains("/sqlite/")) {
```

Comment:
This is less robust than I would like, but the gwc sqlite OperationsRest configuration seems to be fairly permisive with paths (`@RequestMapping("**/sqlite")`) so I'm not quite sure what I can savely do that is more restricted.

Fortunately, the MultipartResolver doesn't seem to do anything when there is not multipart content, so we should be safe even if this gets applied where it shouldn't (for instance, if someone has a resource directory named `sqlite`)